src/chopper_count.cpp

```diff
@@ -12,56 +12,85 @@ namespace chopper::count
-                      "chopper count creates a `[PREFIX].counts` file that contains an estimated kmer count for each "
-                      "user bin which is used in chopper layout. Additionally a directory [PREFIX]_sketches is created "
-                      "and one `.hll` file per user bin is computed. The sketch files improve the layout further. "
+                      "The program creates a `[PREFIX].counts` file that contains an estimated kmer counts for each "
+                      "sequence file. These weights are used in chopper layout to create an index layout file that can "
+                      "be used by the app raptor to build an HIBF. Additionally a directory [PREFIX]_sketches is "
+                      "created that will contain one `.hll` file per sequence file. The sketch files improve the "
+                      "layout computation."
                       "Attention: Expects a prefix, thus file extensions are dropped and if given a path (e.g. /tmp/) "
                       "a default prefix is appended.",
                       seqan3::option_spec::required);
 
+    parser.add_option(config.num_threads,
+                      't', "threads",
+                      "The number of threads to be used for parallel processing.");
+
     parser.add_option(config.column_index_to_cluster,
-                      'c', "column-index",
+                      '\0', "column-index",
                       "The column index by which to cluster. Clustering is not supported at the moment",
                       seqan3::option_spec::hidden);
 
-    parser.add_option(config.num_threads,
-                      't', "threads",
-                      "Number of threads.");
-
+    parser.add_subsection("Parameter tweaking:");
+    // -----------------------------------------------------------------------------------------------------------------
     parser.add_option(config.k,
-                      'k', "kmer-size",
-                      "The kmer size to count minimisers.");
+                      '\0', "kmer-size",
+                      "The kmer size influences the estimated weights. Choosing a kmer size that is too small for your "
+                      "data will result in files appearing more similar than they really are. Likewise a large kmer "
+                      "size might miss out on certain similarities. For dna sequences a kmer size between [?,?] has" // range??
+                      "proven to work well.");
 
     parser.add_option(config.w,
-                      'w', "window-size",
-                      "The window size for minimisers.");
+                      '\0', "window-size",
+                      "Instead of looking at all kmers, one can only store a representative set of kmers, e.g. "
+                      "minimizers. Their number is proportional to that of all kmers in a sequence but the total "
+                      "amount is less, resulting in lower memory consumption (RAM).");
```

Comment:
Oh.. I forgot to think about this when removing the counts. 

1. I had felix check if the estimations work with minimizers and he tried some (w,k)s and said it worked well
2. I think you are right though, since we do not store all kmers now but ONLY compute sketches, and the sketches will have a fixed size, it doesn't matter RAM wise.

So: Is computing minimizers faster then kmer hashing? no right? Then I will delete this option since.

Reply:
I'll run the seqan3 benchmarks and check, but I think k-mers are faster as there is no bookkeeping. Minimisers are faster for the (H)IBF, because less k-mers means less IBF accesses.